implementations/ajv/main.mjs

```diff
@@ -41,7 +41,8 @@ async function validateSchema(schemaPath, instancePath) {
   const startTime = performance.now();
   for (const instance of instances) {
     if (!validate(instance)) {
-      process.exit(1);
+      // XXX Temporarily allow failures
+      // process.exit(1);
```

Comment:
Could we explain why at least? or maybe you forgot to remove it :)

Reply:
@tony-go I left it that way until we made a decision. ajv fails to validate several examples despite the other validators having no problems. I'd be for allowing ajv to fail validation in some cases since other validators while catch any potential errors. We already have to make the argument that ajv is wrong.